corehq/apps/users/management/commands/add_attendance_coordinator_roles.py

```diff
@@ -0,0 +1,41 @@
+from django.core.management.base import BaseCommand
+
+from corehq.apps.users.models_role import UserRole
+from corehq.apps.users.role_utils import UserRolePresets
+from corehq.apps.accounting.models import Subscription
+from corehq.apps.accounting.models import SoftwarePlanEdition
+
+
+class Command(BaseCommand):
+    help = "Adds the Attendance Coordinator role to domains on the pro, advanced and entrprise plans."
+
+    def handle(self, **options):
```

Comment:
Would there be any need to add the role for a single domain only?

Reply:
No, but technically yes. Since we need to test this stuff we will need this role on the test domain, but for that we can just go through the UI and 1. enable the Feature Flag and 2. upgrade to a paid plan. If we do however see the need, we can just update this code, but I highly doubt that we'll need it.